webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -1387,6 +1385,8 @@ public static ByteArrayOutputStream requestsToZip(ServletContext context, String
                     doc.addTitle(globalTitle);
                 }
 
+                // perhaps we should use i18n on the cutout sheet (scramblexofY)
```

Comment:
Could you file an issue for this instead? I don't like tracking work in code comments because you can't have a conversation.

Reply:
Sure!

Reply:
Fixed